tgui/packages/tgui/interfaces/TankDispenser.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2020 @actioninja
+ * Minor changes by Aleksej Komarov
+ * SPDX-License-Identifier: MIT
+ */
+
+import { useBackend } from '../backend';
+import { Button, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+export const TankDispenser = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window
+      width={275}
+      height={105}>
+      <Window.Content>
+        <Section>
+          <LabeledList>
+            <LabeledList.Item
+              label="Plasma"
+              buttons={(
+                <Button
+                  icon={data.plasma ? 'circle' : 'circle-o'}
```

Comment:
FWIW, I don't like the usage of icons here and think this should be conveyed via styling due to it being disabled. If it does that in addition to the icon then I guess I'm okay with it.

Reply:
it does both